set-based-validation/src/main/java/io/axoniq/dev/samples/api/RequestEmailChangeCommand.java

```diff
@@ -0,0 +1,47 @@
+package io.axoniq.dev.samples.api;
+
+import java.util.Objects;
+import java.util.UUID;
+
+import org.axonframework.commandhandling.RoutingKey;
+
+public class RequestEmailChangeCommand {
+    @RoutingKey
```

Comment:
Why are we using `@RoutingKey` instead of `@TargetAggregateIdentifier` here? For clarity, might be worth explaining on a comment/javadoc.